source/game/field/CourseColMgr.hh

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include "egg/math/BoundBox.hh"
+#include "egg/math/Math.hh"
+#include "egg/math/Vector.hh"
+#include "game/field/RKGndCol.hh"
```

Comment:
Are we sure we need `egg/math/Math.hh` in this header?

```suggestion
#include "game/field/RKGndCol.hh"

#include <egg/math/BoundBox.hh>
```

Reply:
It's in here because of the use of `EGG::Mathf::isNonNegative` in `CourseColMgr.cc`. See my other comment regarding the removal of that function, before we proceed with whether or not to remove this header include.